process_jartic_simple.py

```
@@ -0,0 +1,163 @@
+    print(f"Archives found: {len(archives)}")
+    print(f"Date range: {args.start} to {args.end}")
+    
+    # Prepare output
+    storage = DataStorage()
+    timestamp = datetime.now().strftime('%Y%m%d_%H%M%S')
+    output_path = storage.get_processed_dir('jartic') / f"jp_traffic_all_{timestamp}.csv"
+    output_path.parent.mkdir(parents=True, exist_ok=True)
+    
+    # Write header
+    with open(output_path, 'w') as f:
+        f.write("timestamp,location_id,location_name,latitude,longitude,parameter,value,unit,prefecture\n")
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)